### ISSUE TITLE
grynaprzegladarke.pl

### ISSUE THREAD

**Issue by kulfoon** [easylistpolish/easylistpolish#18]
Ads remains / text / placeholders "reklama"

### `http://grynaprzegladarke.pl/gra/elvenar/`

![ss022](https://user-images.githubusercontent.com/43844449/48664224-cc1bbb00-ea9b-11e8-80a4-b87c2f642fe9.png)

**Comment by easylistpolish** [comment]
Sign left (extended filter needed)

**Comment by kulfoon** [comment]
Not really, it's fixed in POL: polskie filtry do Adblocka i uBlocka anyway, any by the way, as seen, the owner was of a different opinion than you, he didn't need extended filters, he added just:
`grynaprzegladarke.pl##.text-center.content > .muted.small`

**Comment by kulfoon** [comment]
Can you tell it to him? I would like to see the confrontation of you vs him, to find out who is right.

**Comment by jspenguin2017** [comment]
> This kind of filters (.class1.class2) are also not best for performance

What? Do you have any supporting arguments? 
I'm pretty confident that modern browser engines can easily optimize that.

**Comment by easylistpolish** [comment]
Hey, sorry for late reply. I'm not following closed issues too much.
> Can you tell it to him? I would like to see the confrontation of you vs him, to find out who is right.
I don't see point to confrontate. Our lists has different goals. I focused only on ads, and trying to make my list as short and as simple as possible. Eg. Now I'm taking more time on reviewing old filters to find and delete unnecessary ones on Easylist Polish.

> What? Do you have any supporting evidence?
> I'm pretty confident that modern browser engines can easily optimize that.
I wrote "not best". I meant by that it's of course worse then blocking filter, and minimal worse then simplest hiding filter like example.com##.class. Sure there's not really difference on modern browsers between ##.class and ##.class1.class2. But as I mention in some other issues, there're also users on mobile, so in cases like small sign "reklama" I don't want to put other filter then simplest one. I don't have dedicated mobile list so I decided to don't put specifically dedicated filter for each small sign left after ad.
